inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/management/DynamicWorkloadManagementPage.html

```diff
@@ -17,84 +17,112 @@
+            right: 0;
+            z-index: 1000;
+            display: none;
+            float: left;
+            min-width: 10rem;
+            padding: 0.5rem 0;
+            margin: 0.125rem 0 0;
+            font-size: 1rem;
+            color: #212529;
+            text-align: left;
+            list-style: none;
+            background-color: #fff;
+            background-clip: padding-box;
+            border: 1px solid rgba(0, 0, 0, 0.15);
+            border-radius: 0.25rem;
+        }
+
+    </style>
 </wicket:head>
-</head>
 <body>
-  <wicket:extend>
-    <div class="flex-content flex-v-container flex-gutter">
-      <div class="page-header">
-        <h1>
-          <wicket:container wicket:id="name" />
-        </h1>
-      </div>
-      <div class="flex-content flex-v-container flex-gutter flex-only-internal-gutter">
-        <div class="flex-content card m-2 ">
-          <div class="card-header">
-            <wicket:message key="documentStatus" />
-            <div class="actions">
-              <span class="dropdown" aria-haspopup="true" aria-expanded="false">
-                <button class="btn btn-action btn-secondary dropdown-toggle flex-content"
-                  type="button" data-toggle="dropdown">
-                  <i class="fas fa-filter"></i>&nbsp;
-                  <wicket:message key="Filter" />
-                </button>
-                <div class="dropdown-menu shadow-lg pt-0 pb-0" role="menu" style="min-width: 800px;">
-                  <form wicket:id="searchForm">
-                    <div class="card-header">
-                      <wicket:message key="filterHeader" />
-                    </div>
-                    <div class="card-body">
-                      <div class="form-group form-row">
-                        <label class="col-form-label col-sm-4">
-                          <wicket:message key="userFilterText" />
-                        </label> 
-                        <div class="col-sm-8">
-                          <input wicket:id="userFilter" type="text" class="form-control" />
-                        </div>
+<wicket:extend>
+  <div class="flex-content flex-v-container flex-gutter">
+    <div class="page-header">
+      <h1>
+        <wicket:container wicket:id="name"/>
+      </h1>
+    </div>
+  <div class="flex-content flex-v-container flex-gutter flex-only-internal-gutter">
+    <div class="flex-content card m-2 ">
+      <div class="card-header">
+        <wicket:message key="documentStatus"/>
+          <div class="actions">
+            <span class="dropdown" aria-haspopup="true" aria-expanded="false">
+              <button class="btn btn-action btn-secondary dropdown-toggle flex-content"
+                type="button" data-toggle="dropdown">
+                <i class="fas fa-filter"></i>&nbsp;
+                <wicket:message key="Filter"/>
+              </button>
+              <span class="dropdown-menu shadow-lg pt-0 pb-0" role="menu" style="min-width: 800px;">
```

Comment:
I believe 800px as min-width is too generous. How about using 400px as min-width and allowing to grow relative to the screen size?

Reply:
Tried here also a lot and adjusted some variables. But same as with the comment above, I cannot go below 600px here. Growth also not required

Reply:
We can go with 600px for the moment and if necessary see if/which other options there may be in a future UI update. Thanks!.